pandas/tests/io/test_html.py

```diff
@@ -3,13 +3,19 @@
 import glob
 import os
 import re
+import threading
 import warnings
 
 try:
     from importlib import import_module
 except ImportError:
     import_module = __import__
 
+try:
+    from importlib import reload
```

Comment:
Let's merge these two `try-except` together since both come from Python 2.x.  Also, make a comment about that `importlib` is from Python 2.x.